skimage/transform/radon_transform.py

```diff
@@ -23,10 +9,10 @@
 from warnings import warn
 
 
-__all__ = ["radon", "iradon", "iradon_sart"]
+__all__ = ['radon', 'order_angles_golden_ratio', 'iradon', 'iradon_sart']
 
 
-def radon(image, theta=None, circle=False):
+def radon(image, theta=None, circle=None):
```

Comment:
Probably should raise a deprecation warning if `circle==None`. Same for `iradon`.